### PR TITLE
[BEAM-13877] [Playground] Increase test coverage for the executors package

### PR DESCRIPTION
**Pull request by pavel-avilov**
[[BEAM-13877]](https://issues.apache.org/jira/browse/BEAM-13877)
Add unit tests to increase test coverage for the executors package.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by pavel-avilov** [comment]
@ilya-kozyrev @KhaninArtur

File: playground/backend/internal/executors/executor_builder_test.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+	type fields struct {
+		actions []handler
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   *ValidatorBuilder
+	}{
+		{
+			name:   "Get RunBuilder with empty actions",
```
Comment (KhaninArtur) [inline playground/backend/internal/executors/executor_builder_test.go:106]:
```suggestion
			name:   "Get ValidatorBuilder with empty actions",
```

File: playground/backend/internal/executors/executor_builder_test.go [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   *CompileBuilder
+	}{
+		{
+			name:   "Get CompileBuilder with empty actions",
+			fields: fields{actions: []handler{}},
+			want:   &CompileBuilder{ExecutorBuilder{actions: []handler{}}},
```
Comment (KhaninArtur) [inline playground/backend/internal/executors/executor_builder_test.go:54]: Can we add cases when `actions` are not empty for all these tests?

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: playground/backend/internal/executors/executor_builder_test.go (hunk 2 above)
Comment (pavel-avilov) [reply playground/backend/internal/executors/executor_builder_test.go:73]: Done

**Review by daria-malkova — APPROVED** [review]
LGTM

**Comment by pabloem** [comment]
lgtm thanks!

**Pull request merged**